### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Zeid69** [datasciencecompleteguide/Machine-Learning#31] [Q&A]
![IMG_20211101_210918](https://user-images.githubusercontent.com/93650929/140045893-bbebcf77-c0cf-46ba-ba84-ec3cccddf837.jpg)

import cv2 as cv
import numpy as np 3

image = cv.imread("butterfly.jpg", 1)
if image is None:
print('Could not open or find the image: ')
exit(0)

try:
bright = int(input('* Enter the brightness value [0-255]: '))
except ValueError:

print('Error, not a number')

new_image = np.zeros(image.shape, image.dtype)

for y in range (image.shape [0]):
for x in range(image.shape[1]): 
for z in range(image.shape [2]):
newval image[y,x,z] + bright =

if newVal > 255:
newVal = 255
new_image[y,x,z] = newVal

cv.imshow(Original Image', image)
cv.imshow("Brightness Image', new_image)
cv.waitKey()

**Comment by datasciencecompleteguide** [comment]
> ![IMG_20211101_210918](https://user-images.githubusercontent.com/93650929/140045893-bbebcf77-c0cf-46ba-ba84-ec3cccddf837.jpg)
> 
> import cv2 as cv import numpy as np 3
> 
> image = cv.imread("butterfly.jpg", 1) if image is None: print('Could not open or find the image: ') exit(0)
> 
> try: bright = int(input('* Enter the brightness value [0-255]: ')) except ValueError:
> 
> print('Error, not a number')
> 
> new_image = np.zeros(image.shape, image.dtype)
> 
> for y in range (image.shape [0]): for x in range(image.shape[1]): for z in range(image.shape [2]): newval image[y,x,z] + bright =
> 
> if newVal > 255: newVal = 255 new_image[y,x,z] = newVal
> 
> cv.imshow(Original Image', image) cv.imshow("Brightness Image', new_image) cv.waitKey()


Hey,
What happen??
I think you forgot to tell the error that you wanted to ask!!
